docs/services/kafka/upgrade.md

```diff
@@ -0,0 +1,50 @@
+# Upgrading Kafka
```

Comment:
Some things to check here:
1. Will running the `deploy_kafka.yml` playbook restart Kafka when it updates the config?
  - My read of the playbook is that it won't but I'm not 100% sure.
  - For the upgrade process we need to restart Kafka after the configs and binaries have changed

2. Not sure if step 5 & 6 need to be separate or if they can be combined.

Reply:
it doesn't do restart , we should add it that seems like right thing to do ?

Reply:
Yes, I think we should add in handlers to restart the services